src/main/java/io/cdap/delta/bigquery/BigQueryTarget.java

```diff
@@ -200,9 +200,10 @@ private static String stringifyPipelineId(DeltaPipelineId pipelineId) {
     private String stagingBucket;
 
     @Nullable
-    @Description("The location where the staging gcs bucket will get created. "
-      + "This value is ignored if the bucket already exists.")
-    protected String stagingBucketLocation;
+    @Description("The location where the Bigquery dataset and GCS staging bucket will be created. Bigquery dataset " +
+      "and GCS Staging Bucket must be in the same location. This value is ignored if an existing GCS bucket is " +
+      "specified as staging bucket and the Bigquery dataset will be created in the same location as that bucket")
+    protected String location;
```

Comment:
renaming this is backwards incompatible (for ex: upgrading a replicator will cause the old value to be ignored), is that ok?

If not, there needs to be a little more complicated logic where this new one is introduced, stagingBucketLocation uses 
 the 'hidden' widget-type, and both are checked in the backend.

Reply:
good point, then I think we don't need to bother renaming the the variable name.